docs/visual-testing/workflows/review.md

```diff
@@ -124,6 +124,24 @@ There is also an option to reject the snapshot. This will mark the snapshot as "
 
 More information about reviewing and approving diffs can be found in [Subsequent Test Execution Review](#subsequent-test-execution-review).
 
+##### Diff inspection
+
+The Diff Inspection Tool is a valuable utility that helps identify and verify which DOM changes have impacted the UI. It offers a visual representation of the differences between two versions, making it easy to spot modified, added, or removed elements.
+
+It detects changes in:
+- html
+- styles
+- position
+- dimensions
+
+The `captureDom` option is disabled by default. For information on how to enable this option or for examples, please refer to the integration page.
+
+:::note
+To generate the DOM diff, it is important to ensure that both the baseline and snapshot have accurately captured the DOM.
```

Comment:
Is it worth emphasizing here something like "...so you will have to run your tests twice to see DOM diff data" ?

Reply:
I think it shouldn't be a part of this section. Baselines might origin from many builds not just only previous.